src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,25 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/todo';
+
+type Props = {
+  todos: Todo[],
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => {
+        const { user, id } = todo;
+
+        return user && (
```

Comment:
```suggestion
        return (
```
this check could be redundant